ember-cli-build.js

```diff
@@ -3,6 +3,7 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const MergeTrees = require('broccoli-merge-trees');
 const Funnel = require('broccoli-funnel');
 const targets = require('./config/targets');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
Why adding this dependency ? Also how it is related to reduce the bundle sizes of js chunks ?

Reply:
All can be answered if you search the name of the dependency

Reply:
👍